ModelUploader/Program.cs

```diff
@@ -23,11 +24,14 @@ public class Program
 
         private static DigitalTwinsClient client;
         private static string modelPath;
+        private static bool deleteFirst;
 
         private class CliOptions
         {
             [Option('p', "path", Required = true, HelpText = "The path to the on-disk directory holding DTDL models.")]
             public string ModelPath { get; set; }
+            [Option('d', "deletefirst", Required = false, HelpText = "Specify if you want to delete the models first, by default is false")]
```

Comment:
Please add in the readme on how to run it, explain that you have to run first REC models and second Willow models.
Also give an example in the readme on how to run delete, e.g. 
ModelUploader\bin\Debug\netcoreapp3.1\ModelUploader -d -p Ontology\

Reply:
Updated Readme